src/components/CadenceEditor/ControlPanel/index.tsx

```diff
@@ -398,54 +397,55 @@ const ControlPanel: React.FC<ControlPanelProps> = (props) => {
       <div ref={constraintsRef} className="constraints" />
       <MotionBox dragConstraints={constraintsRef}>
         <HoverPanel minWidth={showPrompt ? 'min-content' : '300px'}>
-          <Hidable hidden={!validCode}>
-            {list.length > 0 && (
-              <>
-                <ArgumentsTitle
-                  type={type}
-                  errors={numberOfErrors}
-                  expanded={expanded}
-                  setExpanded={setExpanded}
-                />
-                <ArgumentsList
-                  list={list}
-                  errors={errors}
-                  hidden={!expanded}
-                  onChange={(name, value) => {
-                    let key = name.toString();
-                    let newValue = { ...values, [key]: value };
-                    setValue(newValue);
-                  }}
-                />
-              </>
-            )}
-            {needSigners && (
-              <SignersPanel
-                maxSelection={signers}
-                selected={selected}
-                updateSelectedAccounts={updateSelectedAccounts}
+          {list.length > 0 && (
+            <>
+              <ArgumentsTitle
+                type={type}
+                errors={numberOfErrors}
+                expanded={expanded}
+                setExpanded={setExpanded}
               />
-            )}
-          </Hidable>
-
-          <ErrorsList list={problems.error} actions={actions} />
+              <ArgumentsList
+                list={list}
+                errors={errors}
+                hidden={!expanded}
+                onChange={(name, value) => {
+                  let key = name.toString();
+                  let newValue = { ...values, [key]: value };
+                  setValue(newValue);
+                }}
+              />
+            </>
+          )}
+          {needSigners && (
+            <SignersPanel
+              maxSelection={signers}
+              selected={selected}
+              updateSelectedAccounts={updateSelectedAccounts}
+            />
+          )}
+
+
+          {/*<ErrorsList list={problems.error} actions={actions} />*/}
```

Comment:
commented out code?

Reply:
leaving it commented for potential future work